README.md

```diff
@@ -1,110 +1,162 @@
 
 ```
 pre-commit autoupdate && pre-commit install
 ```
 
-This will install / configure the pre-commit git hooks, if it detects an issue while committing it will produce an output like the following:
+This will install and configure the pre-commit Git hooks. 
 
-```
- git commit -a
-check json...........................................(no files to check)Skipped
-fix end of files.........................................................Passed
-trim trailing whitespace.................................................Passed
-detect aws credentials...................................................Passed
-detect private key.......................................................Passed
-AWS CloudFormation Linter................................................Failed
-- hook id: cfn-python-lint
-- exit code: 4
-W3011 Both UpdateReplacePolicy and DeletionPolicy are needed to protect Resources/PublicHostedZone from deletion
-core/deploy/dns-zones/template.yaml:20:3
-Checkov..............................................(no files to check)Skipped
-- hook id: checkov
-```
+## GitHub tools and workflows
+
+### GitHub CODEOWNERS
+
+You can use a [GitHub CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file to define individuals or teams that are responsible for code in a repository. Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
+
+You can find the [`CODEOWNERS` file for ipv-core-front](https://github.com/govuk-one-login/ipv-core-front/blob/main/CODEOWNERS) in the root.
+
+### Uploading assets to Amazon CloudFront
+
+If you add a new MIME type to either the `/assets` or `/public` folders, you must also add it to the CloudFront function in that repository.
 
-### Assets upload to  CDN
-There is a step in the github workflow secure-post-merge.yml which pushes a zip of the /public
-and /assets folders to an S3 bucket, this is then decompressed and put into another bucket
-which is behind a cloudfront distribution. The infra for this is in:
-[https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets](https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets)
-If you add a new mimetype to the /assets or /public folders it will need to be added to the cloudfront
-function in that repo.
+This is because there is a step in the `secure-post-merge.yml` GitHub workflow that pushes a ZIP archive of the `/public` and `/assets` folders to an Amazon S3 bucket. Following this, the archive is decompressed and transferred to a separate bucket situated behind a CloudFront distribution. The infrastructure for this is hosted at [https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets](https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets).
```

Comment:
```suggestion
This is because there is a step in the `secure-post-merge.yml` GitHub workflow that pushes a ZIP archive of the `/public` and `/assets` folders to an Amazon S3 bucket. The archive is then decompressed and transferred to a separate bucket behind a CloudFront distribution, which is hosted at [https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets](https://github.com/alphagov/di-ipv-core-common-infra/tree/main/cloudformation/upload-assets).
```
Feels like this should be a continuation of the first paragraph. And a couple of minor wording suggestions.